.jules/learnings.md

```
@@ -0,0 +1,4 @@
+## 2026-05-15 - Fixing Missing and Invalid Javadocs
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)